cmd/tempo-serverless/handler.go

```diff
@@ -196,6 +198,7 @@ func setDecodeHooks(c *mapstructure.DecoderConfig) {
 		mapstructure.StringToTimeDurationHookFunc(),
 		mapstructure.StringToSliceHookFunc(","),
 		stringToFlagExt(),
+		mapstructure.StringToTimeHookFunc(time.RFC3339),
```

Comment:
Do you have an example of a config that this allows to be parsed?  Would it be to set timestamps on the GCS objects?

Reply:
I don't yet, but I will be working on integration tests soon that should surface this elsewhere.  Though, I don't think we're using viper elsewhere so this specifically shouldn't be an issue after this change.  This came up because of the marshal<>unmarshal round tripping that the viper `loadConfig()` does for the serverless stuff gave a default value for the `time.Time`, which was then confusing the mapstructure pkg.

Reply:
Do we have any config fields that are of type time? I'm not sure we need this.

Reply:
https://github.com/googleapis/google-cloud-go/blob/main/storage/storage.go#L1391 for reference.